### ISSUE TITLE
Neither login nor signup through Google works for me

### ISSUE THREAD

**Issue by yarikoptic** [neuroscout/neuroscout#866]
It seems to do something useful, but the original dialog on neuroscout never disappears and web ui never acknowledges that I am logged in (which I am not)

**Comment by adelavega** [comment]
Looking into it. Do you have any privacy extensions active per chance?

**Comment by adelavega** [comment]
Yep, just made a testneuroscout account on Google and was able to successfully sign up and sign in. 

Firefox 84.0.2 on Ubuntu 20.04

**Comment by yarikoptic** [comment]
right... it was due to "Block third-party cookies" in chromium.  Typically I just wait for a site to announce that I am doing that, enable temporarily, let them pollute with more cookies, and then disable again.  So something like that ideally should be done here as well

**Comment by adelavega** [comment]
A bit ironic Chromium is blocking Google sign on eh? Can you white list certain websites? That's what I've done.

I change the title of this issue, as you're not the first to have this issue. Maybe a warning or error catching of some sort would help
